tasks/prebootstrap.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Set history timestamps and size
+  blockinfile:
+    path: /etc/bash.bashrc
+    block: |
+      HISTSIZE=5000
```

Comment:
@speedlight overall looks good, I just wanted if we could make it more parameterizable:

```yaml
-- defaults/main.yml
sb_debian_base_bash_hist_size: 5000
sb_debian_base_bash_hist_file_size: 3000
sb_debian_base_bash_hist_time_format: '%c%t'
```

Then you should be able to refactor this task a little bit:

```yaml
block:
  path: /etc/bash.bashrc
  block: |
    HISTSIZE={{ sb_debian_base_bash_hist_size }}
    ...
```

Reply:
Additionally, maybe it would be a good idea to make this feature optional:

```yaml
-- defaults/main.yml
sb_debian_base_bash_hist: no
```

```yaml
blockinfile:
  path: /etc/bash.bashrc
  block: ...
when: "{{ sb_debian_base_bash_hist | bool }}"
```

That way we let the consumer decide whether they want to change bash history settings or not. What do you think?

Reply:
Nice points @sestrella thanks! I will do the changes to achieve this.